### PR TITLE
Fixed writing of homee-Mode and moved to current homee-api

### PR DESCRIPTION
**Pull request by Seraphis411**
my first pull request, so feel free to comment / correct :)

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -380,8 +380,7 @@ adapter.on('ready', () => {
 }*/
 
 function initNodes(nodes) {
-    adapter.log.info('initialize ' + nodes.length + ' nodes');
-    adapter.log.silly('Received NODES: ' + JSON.stringify(nodes));
```
Comment (Apollon77) [inline main.js]: ehy you removed the silly logging here? It's good for debugging

File: main.js [hunk 2]
```diff
@@ -104,11 +104,11 @@ adapter.on('stateChange', (id, state) => {
...
     }
-
-    let nodeId = parseInt(idParts[0].split('-')[1], 10);
-    if (nodeId === 0) nodeId = -1;
+	
+    let nodeId = parseInt(idParts[0].split('-')[1], 10);    
     const attributeId = parseInt(idParts[1].split('-')[1], 10);
-    const lookupId = nodeId + '.' + attributeId;
+    const lookupId =  nodeId + '.' + attributeId;
+	if (nodeId === 0) nodeId = -1;
```
Comment (Apollon77) [inline main.js]: Please fix the two too much spaces :)

File: main.js [hunk 3]
```diff
@@ -418,13 +417,9 @@ function initHomeegram(homeegram) {
     adapter.getObject(nodeId, (err, obj) => {
         if (!err && obj) {
             adapter.extendObject(nodeId, {
-                type: 'state',
+                type: 'device',
```
Comment (Apollon77) [inline main.js]: why you changed this?

File: main.js (hunk 1 above)
Comment (Seraphis411) [reply main.js]: Readded it again now. Seemed broken at first and gave me a type error regarding circular references, but is working now.

File: main.js (hunk 2 above)
Comment (Seraphis411) [reply main.js]: done, seems tab-stops aren't the way to go here..

File: main.js (hunk 3 above)
Comment (Seraphis411) [reply main.js]: having a state with button-functionality with states beneath it was confusing for me when i started to use this adapter. seems more "logical" to me this way. now only .active / .play  / .playing are states and the homeegramm-id is the device housing those.
besides, the button didn't even work.. 😝

**Comment by Apollon77** [comment]
SOrry that it took that long, will check and merge soon

**Comment by Apollon77** [comment]
@Seraphis411 could you update to latest api and points? Then I can take that very fast.

File: main.js (hunk 1 above)
Comment (Apollon77) [reply main.js]: I will change that back because the button in admin is not that relevant ... but setStates shuld work and it is needed for legacy support

**Pull request merged**